File: ejercicio1.py
import time

# Medir el tiempo de ejecución
start_time = time.time()

# Ejercicio 1: Suma y Promedio
valores = []
for i in range(10):
    valor = float(input(f"Ingrese el valor {i+1}: "))
    valores.append(valor)

suma = sum(valores)
promedio = suma / len(valores)

print(f"Suma: {suma}, Promedio: {promedio}")

# Fin del tiempo de ejecución
end_time = time.time()
print(f"Tiempo de ejecución en Python: {end_time - start_time} segundos")

File: ejercicio1.cpp
#include <iostream>
#include <vector>
#include <chrono>

using namespace std;

int main() {
    auto start = chrono::high_resolution_clock::now();
    
    vector<double> valores;
    double valor, suma = 0;
    
    for (int i = 0; i < 10; i++) {
        cout << "Ingrese el valor " << i + 1 << ": ";
        cin >> valor;
        valores.push_back(valor);
        suma += valor;
    }
    
    double promedio = suma / valores.size();
    cout << "Suma: " << suma << ", Promedio: " << promedio << endl;

    auto end = chrono::high_resolution_clock::now();
    chrono::duration<double> tiempo = end - start;
    cout << "Tiempo de ejecución en C++: " << tiempo.count() << " segundos" << endl;

    return 0;
}
